cmd/podman/images/list.go

```diff
@@ -128,10 +129,27 @@ func images(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	imgs, err := sortImages(summaries)
+	reference := ""
+	for _, filter := range listOptions.Filter {
+		if strings.HasPrefix(filter, "reference=") {
+			reference = strings.TrimPrefix(filter, "reference=")
```

Comment:
doesn't the regular filter code allow more than one reference? SO I think this needs to handle all not just the first match.

Also you should use CutPrefix()